src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -147,7 +150,7 @@ def get_by_username(
         pass
 
     @abstractmethod
-    def get_from_group(self, group: Group) -> List[RpcUser]:
+    def get_from_group(self, group: "Group") -> List[RpcUser]:
```

Comment:
A few straggler forward references to classes other than RPC data models, such as `"Group"` here, may not be a problem. If they are, my preferred fix would be to prevent the circular import by, for example, going to `sentry.models.group` and moving `from sentry.services.hybrid_cloud.user import RpcUser` to a local function context. But I can take care of that when we get there.